webrtc.html

```diff
@@ -16732,6 +16732,29 @@ <h4>
                   double rounded to the tenth decimal place.
                 </td>
               </tr>
+              <tr id="def-constraint-channelCount">
+                <th data-tests="RTCRtpEncodingParameters-codec-opus-stereo.https.html">
+                  channelCount
+                </th>
+                <td>
+                  {{ConstrainULong}}
+                </td>
+                <td>
+                  As a setting, this is the number of {{RTCRtpCodec/channels}}
+                  that the decoder produced when the last audio frame was
+                  decoded. The number of channels that the decoder produces MUST
+                  reflect the number of channels configured when the local SDP
+                  was set. Before any frames have been decoded, this setting
+                  MUST NOT [=map/exist=].
```

Comment:
I don't believe the constrainable pattern allows absence. Is there a precedent in the spec for a setting to not exist and come into being later? I don't think we should allow that. I've filed https://github.com/w3c/webrtc-pc/issues/3045

```suggestion
                  was set.
```

Reply:
> Is there a precedent in the spec for a setting to not exist and come into being later?

Yes this is how remote track settings already work in the video frames (width, height and frameRate) so I thought we should be consistent with audio frames.